docs/gitbook/tutorials/prometheus-operator.md

```diff
@@ -0,0 +1,172 @@
+# Flagger with Prometheus Operator
+
+This guide will show you how to use Flagger and Prometheus Operator.  
+This guide will handle only Blue/Green Deployment with podinfo application
+
+## Prerequisites
+
+Flagger and Prometheus Operator requires a Kubernetes cluster **v1.11** or newer
+
+Install Prometheus-Operator with Helm v3:
+
+```bash
+helm repo add stable https://kubernetes-charts.storage.googleapis.com
+helm repo update
+kubectl create ns prometheus
+helm upgrade -i prometheus stable/prometheus-operator \
```

Comment:
Please use [fullnameOverride](https://github.com/helm/charts/blob/master/stable/prometheus-operator/values.yaml#L15) and set the namespace to `monitoring` so that the URL will look better.